pkgs/applications/networking/instant-messengers/zulip/default.nix

```diff
@@ -1,36 +1,70 @@
 { lib
-, fetchurl
-, appimageTools
+, fetchFromGitHub
+, buildNpmPackage
+, electron
```

Comment:
We need to pin this to the correct major version, to be bumped during updates. Electron makes breaking API changes in major versions fairly regularly.

Indeed, this doesn’t work with the current `electron == electron_28`. The inner webviews fail to load with errors like `Message REMOTE_RENDERER_CALLBACK sent by unexpected WebContents (undefined)`.

`electron_25` is EOL now, so I’ll work on making a new upstream release with a newer Electron. (At the time of the last release, we were blocked by electron/electron#39775.)

Reply:
Ah, I thought I tested electron 28, but realised what happens is:
* I tested the package with an overlay, and since I'm on 23.11, it uses electron 27.
* I then prepared the PR and do `nix run .#zulip` -- this also works, but actually since now I have a zulip running in background it just opened the window and didn't launch a new instance.

I closed running zulip w/ electron 27 and relaunch electron 28 and indeed it's not compatible. Will pin to 27.